src/js/_store/modules/articles.js

```diff
@@ -115,20 +134,55 @@ export default {
     displayDoneMessage(state, payload = { message: '成功しました' }) {
       state.doneMessage = payload.message;
     },
+    moveArticlePage(state, { pageAll, pageId }) {
+      state.articleList = [...pageAll.articles];
+      const currentPageId = parseInt(pageId, 10);
+      const lastPageObject = {
+        lastPage: state.pages.lastPage,
+      };
+      const lastPageNo = lastPageObject.lastPage;
+      if (pageId <= 4) {
+        state.pages.firstPage = 1;
+        state.pages.secondPage = 2;
+        state.pages.thirdPage = 3;
+        state.pages.fourthPage = 4;
+        state.pages.fifthPage = 5;
+        state.pages.sixthPage = 6;
+      } else if (pageId >= lastPageNo - 3) {
+        state.pages.secondPage = lastPageNo - 5;
+        state.pages.thirdPage = lastPageNo - 4;
+        state.pages.fourthPage = lastPageNo - 3;
+        state.pages.fifthPage = lastPageNo - 2;
+        state.pages.sixthPage = lastPageNo - 1;
+      } else {
+        state.pages.secondPage = currentPageId - 2;
+        state.pages.thirdPage = currentPageId - 1;
+        state.pages.fourthPage = currentPageId;
+        state.pages.fifthPage = currentPageId + 1;
+        state.pages.sixthPage = currentPageId + 2;
+      }
+    },
+    reflectPage(state, payload) {
+      state.pages.lastPage = payload;
+    },
   },
   actions: {
     initPostArticle({ commit }) {
       commit('initPostArticle');
     },
-    getAllArticles({ commit, rootGetters }) {
+    getAllArticles({ commit, rootGetters }, pageId) {
+      commit('clearMessage');
       axios(rootGetters['auth/token'])({
         method: 'GET',
-        url: '/article',
+        url: `http://api.wiki.gizumo-inc.work/api/article?page=${pageId}`,
```

Comment:
[nits]
`http://api.wiki.gizumo-inc.work/api`はなくても良いかなと思います！

```suggestion
        url: `/article?page=${pageId}`,
```

Reply:
URLの記述不要な部分については削除いたしました！[6aa36b4](https://github.com/gizumo-education/gizumo_wiki_lesson/pull/137/commits/6aa36b4366d1573fbc41940feb0888d90df8a449)